src/sections/subscribe/subscribe.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Row, Col } from "../../reusecore/Layout";
+import SectionTitle from "../../reusecore/SectionTitle";
+import Button from "../../reusecore/Button";
+import SubscribeWrapper from "./subscribe.style";
+
+const SubscribeSection = () => {
+  return (
+    <SubscribeWrapper>
+      <div className="parentcard">
+        <form name="contactform" method="post" action="https://calcotestudios.us15.list-manage.com/subscribe/post?u=6b50be5aea3dfe1fd4c041d80&amp;id=6bb65defeb">   
+          <Row>
+            <Col md={6}>
+              <SectionTitle className="section-title" UniWidth="100%">
+                <h2><span className="meshy">Stay meshy </span>and subscribe!</h2>
+              </SectionTitle>
+            </Col>
+            <Col md={6}>
+              <Row className="email-cont">
+                <Col md={6} xs={12} >
+                  <input className="inputrow subscribe-email" type="email" placeholder="Email Address" name="EMAIL" id="mce-EMAIL" required />
+                </Col>  
+                <div>
```

Comment:
There is some responsiveness issue probably because this is not inside a `Col` 
![Screenshot from 2021-01-31 02-30-00](https://user-images.githubusercontent.com/42486853/106368000-74ac5180-636c-11eb-8c6c-09fd0d5b12aa.png)

Reply:
Fixed!